web/src/components/TimeSlider.test.ts

```diff
@@ -2,8 +2,8 @@ import { COLORS, getThumbIcon, getTrackBackground } from './TimeSlider';
 
 describe('getTrackBackground', () => {
   it('returns the day color when no sets are provided', () => {
-    expect(getTrackBackground(false, 10)).toEqual(COLORS.light.day);
-    expect(getTrackBackground(true, 10)).toEqual(COLORS.dark.day);
+    expect(getTrackBackground(false, 10)).to.eq(COLORS.light.day);
+    expect(getTrackBackground(true, 10)).to.eq(COLORS.dark.day);
```

Comment:
An alternative to changing all of these would be to import all the functions from vitest directly in all the tests, then both syntaxes would be valid as far as `tsc` is concerned.